sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_get_manual_validation_info_async.py

```diff
@@ -0,0 +1,97 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_get_manual_validation_info_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to output the information that will help with manually
+    validating your output from recognize custom forms.
+USAGE:
+    python sample_get_manual_validation_info_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) CUSTOM_TRAINED_MODEL_ID - the ID of your custom trained model
+"""
+
+import os
+import asyncio
+from pathlib import Path
+
+
+class GetValidationInfoSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    model_id = os.environ["CUSTOM_TRAINED_MODEL_ID"]
+
+    async def get_manual_validation_info_from_recognize_custom_forms(self):
+        # the sample forms are located in this file's parent's parent's files.
+        path_to_sample_forms = Path(__file__).parent.parent.absolute() / Path("sample_forms/forms/Form_1.jpg")
+        # [START create_form_recognizer_client_async]
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer.aio import FormRecognizerClient
+
+        form_recognizer_client = FormRecognizerClient(
+            endpoint=self.endpoint, credential=AzureKeyCredential(self.key)
+        )
+        # [END create_form_recognizer_client_async]
+        async with form_recognizer_client:
+
+            # The form you are recognizing must be of the same type as the forms the custom model was trained on
+            with open(path_to_sample_forms, "rb") as f:
+                forms = await form_recognizer_client.recognize_custom_forms(
+                    model_id=self.model_id, stream=f.read(), include_text_content=True
+                )
+
+            for idx, form in enumerate(forms):
+                print("--------RECOGNIZING FORM #{}--------".format(idx))
+                print("Form has type {}".format(form.form_type))
+                for label, field in form.fields.items():
+                    # each field is of type FormField
+                    # The value of the field can also be a FormField, or a list of FormFields
+                    # In our sample, it is not.
+                    print("Field '{}' has value '{}' based on '{}' within bounding box '{}', with a confidence score of {}".format(
```

Comment:
If the purpose of this sample is to call out form field geometry that can be used to build UI, it would be good to highlight that here.

Reply:
I think we are exposing all of the form field geometry we have (bounding boxes), so people using this to build UI should be covered